components/org.wso2.identity.outbound.adapter.websubhub/src/main/java/org/wso2/identity/outbound/adapter/websubhub/util/WebSubHubAdapterUtil.java

```diff
@@ -209,13 +216,15 @@ public void completed(final HttpResponse response) {
 
             @Override
             public void failed(final Exception ex) {
-
+                WebSubHubCorrelationLogUtils.triggerCorrelationLogForResponse(request, requestStartTime,
+                        RequestStatus.FAILED.getStatus(), ex.getMessage());
```

Comment:
`ex.getMessage()`
If possible shall we check whether we can use some other parameter to get failure reason? While analyzing the logs noticed this was `null`. (Could be the hub not sending any failure reason as well)

Reply:
When trying to reproduce this locally, didn't find such scenarios where the `ex.getMessage()` returns null.